### ISSUE TITLE
Feature to search issues by tags for candidates 

### ISSUE THREAD

**Issue by Paarmita** [publiclab/community-toolbox#32]
In candidates, we can search for issues by tags by making a dropdown of all the tags. 
At present, when we click on tags it opens a JSON data.

**Comment by jywarren** [comment]
This would be great.

Actually I'm really interested in a display of who's completed a first
timers only issue in the past week... Can you think of a way to produce
this list?

**Comment by Paarmita** [comment]
@jywarren It means to filter out first-timer issues contributor by week?

**Comment by jywarren** [comment]
Yes, and show a list of them maybe above the main Contributors listing?
Called "New contributors this week" perhaps?

**Comment by Rishabh570** [comment]
@jywarren I can open a follow-up PR for this once I complete making the MVC thing...like currently, we see recent contributors last week & last month, we can do something to show recent contributors last week for `fto` only...(data we get from Github API gives us a big JSON, that would help prob)

**Comment by Rishabh570** [comment]
@jywarren I can open a follow-up PR for this once I complete making the MVC thing...like currently, we see recent contributors last week & last month, we can do something to show recent contributors last week for `fto` only...(data we get from Github API gives us a big JSON, that would help prob)
